source/includes/2016-03-05/_disputes.md

```diff
@@ -1162,6 +1162,67 @@ chargehound.disputes.submit("dp_123",
 
 In order to submit a Braintree dispute, you will also need to attach the Braintree transaction id using the `charge` parameter. 
 
+## Stripe listing needs response
```

Comment:
I think this should be in the integration section for responding to backlog. It should explain that to make sure you get disputes and retrievals to use them both,